### DISCUSSION TITLE
✨ Add govulncheck to pipeline

### DISCUSSION THREAD

**Discussion by marcosnils** [dagger/dagger#12066] [Ideas]
### What are you trying to do?

Add govulncheck to CI pipeline

https://go.dev/blog/vuln

**Comment by TomChv** [comment]
Done through trivy there: https://github.com/dagger/dagger/blob/eaed4d2e767973023e6d178b8bd77c09d42cce3e/.dagger/engine.go#L410

I'm closing this issue, please reopen it if it was a mistake

**Comment by jedevc** [comment]
The trivy `vuln` scanner (https://trivy.dev/latest/docs/scanner/vulnerability/) is not the same as govulncheck (a core go tool).

We should ideally do both.
